### PR TITLE
Custom build executables

### LINKED ISSUE #35: Recognize blender builds with custom names, like BforArtists
Attempted to add [Bforartist ver 3.6.1](https://www.bforartists.de/) into my custom folder, but the launcher isn't recognizing the folder.

As an aside, I did try making a custom .blinfo for the folder, but still not appearance

image

image

### PR DESCRIPTION
**Pull request by zeptofine**
sorta fixes #35 
video demonstration

### CODE REVIEW

**Comment by Victor-IX** [comment]
I added a platform check to only filter .exe files on Windows to refine the use choice
Do you think it's possible to have the user input for the executable name not case sensitive?

**Comment by zeptofine** [comment]
paths on windows aren't case sensitive, yeah? it should be just as easy as a flag. I'll look into it

File: source/windows/custom_build_dialog_window.py [hunk 1]
```diff
@@ -105,6 +105,8 @@ def __init__(
                 str(file.relative_to(path)) for file in path.iterdir() if file.is_file() and os.access(file, os.X_OK)
             ]
         completer = QCompleter(executables, self)
+        completer.setCaseSensitivity(Qt.CaseSensitivity.CaseInsensitive)
```
Comment (zeptofine) [inline source/windows/custom_build_dialog_window.py:108]: @Victor-IX Is this what you had in mind?

Comment (Victor-IX) [reply source/windows/custom_build_dialog_window.py:108]: Yep, this is perfect! Now Upper case or Lowercase doesn't matter

**Pull request merged**